src/components/SidebarOld.vue

```diff
@@ -0,0 +1,89 @@
+<template>
```

Comment:
This component is not being used anywhere. If this is supposed to be a backup of the old code, it might be better to link the file in GitHub, or comment it out on the Sidebar component?

Reply:
Okay, I will just remove the file and create link